addons/l10n_sa_edi_pos/static/src/js/pos_models.js

```diff
@@ -0,0 +1,15 @@
+odoo.define("l10n_sa_edi_pos.models", function (require) {
+    "use strict";
+
+    const models = require("point_of_sale.models");
+    const _super_order = models.Order.prototype;
+
+    models.Order = models.Order.extend({
+        initialize: function () {
+            _super_order.initialize.apply(this, arguments);
+            if (this.pos.company.country.code === 'SA') {
```

Comment:
```suggestion
            if (this.pos.company.country && this.pos.company.country.code === 'SA') {
```
I think country can be falsy here. That is just to prevent crash.